app/helpers/application_helper.rb

```diff
@@ -1,39 +1,11 @@
 module ApplicationHelper
-  ##
-  # Returns a string build from the start and end date of the given conference.
-  #
-  # If the conference is only one day long
-  # * %B %d %Y (January 17 2014)
-  # If the conference starts and ends in the same month and year
-  # * %B %d - %d, %Y (January 17 - 21 2014)
-  # If the conference ends in another month but in the same year
-  # * %B %d - %B %d, %Y (January 31 - February 02 2014)
-  # All other cases
-  # * %B %d, %Y - %B %d, %Y (December 30, 2013 - January 02, 2014)
-  def date_string(start_date, end_date)
-    startstr = 'Unknown - '
-    endstr = 'Unknown'
-    # When the conference is in the same month
-    if start_date.month == end_date.month && start_date.year == end_date.year
-      if start_date.day == end_date.day
-        startstr = start_date.strftime('%B %d')
-        endstr = end_date.strftime(' %Y')
-      else
-        startstr = start_date.strftime('%B %d - ')
-        endstr = end_date.strftime('%d, %Y')
-      end
-    elsif start_date.month != end_date.month && start_date.year == end_date.year
-      startstr = start_date.strftime('%B %d - ')
-      endstr = end_date.strftime('%B %d, %Y')
-    else
-      startstr = start_date.strftime('%B %d, %Y - ')
-      endstr = end_date.strftime('%B %d, %Y')
-    end
-
-    result = startstr + endstr
-    result
-  end
-
+  # Including custom made helpers
+  include DateTimeHelper
+  include FormatHelper
+  include EventsHelper
+  include UsersHelper
+  include PathsHelper
+  include ChangeDescriptionHelper
```

Comment:
Views donot require custom helpers to be included in application helper but if we are using a helper function in different controller then we have to include that helper ?